chia/consensus/default_constants.py

```diff
@@ -42,10 +42,10 @@
     "MEMPOOL_BLOCK_BUFFER": 150,
     # Max coin amount, fits into 64 bits
     "MAX_COIN_AMOUNT": uint64((1 << 64) - 1),
-    # 1 vByte = 108 clvm cost units
-    "CLVM_COST_RATIO_CONSTANT": 108,
     # Max block cost in clvm cost units
-    "MAX_BLOCK_COST_CLVM": 40000000,  # Based on arvid analysis
+    "MAX_BLOCK_COST_CLVM": 10860254871,
```

Comment:
Might want to make this look rounder, just so it's easier to remember. Maybe `10900000000`

Reply:
Or even `11000000000`

Reply:
yeah 11 billion is easy to remember and to talk about